its_the_way.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+def start_with_deep(path: str):
```

Comment:
add a return type - find what it means in here 
https://docs.python.org/3/library/typing.html